### PR TITLE
 create splash screen

### PR DESCRIPTION
**Pull request by meoconlontonbebong**
link redmine : https://edu-redmine.framgia.vn/issues/6799 
create splash screen 
image

### CODE REVIEW

**Comment by meoconlontonbebong** [comment]
ready

File: app/src/main/java/com/example/moviedb_37/screen/home/HomeFragment.java [hunk 1]
```diff
@@ -54,21 +54,26 @@ private void setupAdapters() {
         mPopularAdapter = new CategoryAdapter(new ArrayList<Movie>(0));
         mPopularAdapter.setItemClickListener(this);
         popularRecycler.setAdapter(mPopularAdapter);
+        popularRecycler.setNestedScrollingEnabled(false);
+
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/example/moviedb_37/screen/home/HomeFragment.java:58]: remove bl

File: app/src/main/java/com/example/moviedb_37/screen/movies/ListMovieCategoryViewModel.java [hunk 2]
```diff
@@ -139,6 +145,7 @@ private void loadMoviesByGenre() {
                 .subscribe(movies -> moviesObservable.addAll(movies),
                         throwable -> handleError(throwable.getMessage()));
         isLoadMore.set(false);
+        isLoadingSuccess.set(true);
         mCompositeDisposable.add(disposable);
     }
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/example/moviedb_37/screen/movies/ListMovieCategoryViewModel.java:151]: remove bl

**Comment by meoconlontonbebong** [comment]
ready

**Pull request merged**